src/components/sidebar/sidebar.css

```diff
@@ -44,8 +44,9 @@
 }
 
 .sidebar-title {
-  font-size: 12px;
+  font-size: 1.2em;
   margin-bottom: .3em;
+  font-weight: 600;
```

Comment:
I don't have a strong opinion but I think that in the "Web" context it should be "500" according to https://design.firefox.com/photon/visuals/typography.html.